kotest-framework/kotest-framework-engine/src/wasmJsMain/kotlin/KotlinTestApi.kt

```diff
@@ -0,0 +1,7 @@
+// The JS launcher set up by the Kotlin Gradle plugin calls this function in addition to `main()`.
+@OptIn(ExperimentalJsExport::class)
+@JsExport
+internal fun startUnitTests() {
+   // The Kotlin compiler would insert test invocations here for kotlin-test.
+   // This mechanism is not used with Kotest.
```

Comment:
Do we have any documentation on how this works in Kotest instead? I keep forgetting. :(

Reply:
I wouldn't call it documentation, but it can be inferred from sources in `kotest-framework-multiplatform-plugin-embeddable-compiler/src/jvmMain/kotlin/io/kotest/framework/multiplatform/embeddablecompiler`, in particular `entry.kt`.